docs/en/table_design/table_types/duplicate_key_table.md

```diff
@@ -6,7 +6,9 @@ displayed_sidebar: "English"
 
 The Duplicate Key table is the default model in StarRocks. If you did not specify a model when you create a table, a Duplicate Key table is created by default.
 
-When you create a Duplicate Key table, you can define a sort key for that table. If the filter conditions contain the sort key columns, StarRocks can quickly filter data from the table to accelerate queries. The Duplicate Key table allows you to append new data to the table. However, it does not allow you to modify existing data in the table.
+When you create a Duplicate Key table, you can define a sort key for that table. If the filter conditions contain the sort key columns, StarRocks can quickly filter data from the table to accelerate queries.
+
+The Duplicate Key table is suitable for scenarios, such as analyzing logs data. It is suitable for scenarios, such as analyzing logs data.
```

Comment:
redundancy -- The Duplicate Key table is suitable for scenarios such as analyzing logs data.